fileAttributeCaching/src/test/kotlin/com/pkware/java/nio/file/fileAttributeCaching/ExpirableCacheTest.kt

```diff
@@ -0,0 +1,98 @@
+package com.pkware.java.nio.file.fileAttributeCaching
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.util.concurrent.TimeUnit
+
+class ExpirableCacheTest {
+
+    private val attributeCache = ExpirableCache<String>(
+        TimeUnit.SECONDS.toMillis(3)
```

Comment:
I reduced it to 1 second, I wanted to keep it at this value at least to simulate what happens when it's used in practice where it will have a 5 second duration.

Reply:
unit tests should never have sleeps in them. it causes the entire thread to stop. the default implementation can call into the clock directly, but this should be pulled behind an interface and mocked for the unit tests. something like https://github.com/trickl/clock, https://github.com/robfletcher/test-clock would be a good place to start looking for reference. The literal actual passage of time is not important, rather that the math is evaluated correctly.